### DISCUSSION TITLE
Special e-mail text for individual products

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretix/pretix#3404] [Q&A]
Hello,

Is there a way that I can add text lines to automated emails for individual products or product variants when the order has been paid?

Or special automated emails which are only sent when ordering certain products or product variants. 

For example, would it be possible to add product metadata to emails?

Background: We have special helper tickets and would like to send additional information automatically for paid orders.

**Comment by raphaelm** [comment]
There currently is no way to achieve this
